airflow/api_connexion/endpoints/backfill_endpoint.py

```diff
@@ -0,0 +1,181 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from __future__ import annotations
+
+import logging
+from typing import TYPE_CHECKING
+
+import pendulum
+from flask import g
+from sqlalchemy import select
+
+from airflow.api_connexion import security
+from airflow.api_connexion.exceptions import Conflict, NotFound, PermissionDenied
+from airflow.api_connexion.schemas.backfill_schema import (
+    BackfillCollection,
+    backfill_collection_schema,
+    backfill_schema,
+)
+from airflow.models.backfill import Backfill
+from airflow.models.serialized_dag import SerializedDagModel
+from airflow.utils import timezone
+from airflow.utils.session import NEW_SESSION, provide_session
+from airflow.www.decorators import action_logging
+from airflow.www.extensions.init_auth_manager import get_auth_manager
+
+if TYPE_CHECKING:
+    from sqlalchemy.orm import Session
+
+    from airflow.api_connexion.types import APIResponse
+
+log = logging.getLogger(__name__)
+
+RESOURCE_EVENT_PREFIX = "dag"
+
+
+@provide_session
+def _create_backfill(
+    *,
+    dag_id,
+    from_date,
+    to_date,
+    max_active_runs=10,
+    reverse=False,
+    dag_run_conf: dict | None = None,
```

Comment:
```suggestion
    dag_id: str,
    from_date: str,
    to_date: str,
    max_active_runs: int,
    reverse: bool,
    dag_run_conf: dict | None,
```
Don't need the defaults - they are already done and passed through to here anyways.